flake.nix

```diff
@@ -91,6 +91,8 @@
       lenovo-legion-16ach6h = import ./lenovo/legion/16ach6h;
       lenovo-legion-16ach6h-hybrid = import ./lenovo/legion/16ach6h/hybrid;
       lenovo-legion-16ach6h-nvidia = import ./lenovo/legion/16ach6h/nvidia;
+      lenovo-legion-16achg6-hybrid = import ./lenovo/legion/16achg6/hybrid;
```

Comment:
Please also add the model to the table in the README.md.

Reply:
Thanks for the feedback. It's done now.